From 0ee305a5e290a66a4c610e57e3aea8bbf66d42d6 Mon Sep 17 00:00:00 2001
From: mohalisad <mohammadalisadraei@gmail.com>
Date: Wed, 24 Nov 2021 20:53:36 +0330
Subject: [PATCH] fix red

---
 main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/atheros/alx/main.c b/drivers/net/ethernet/atheros/alx/main.c
index 43e5f84..8417945 100644
--- a/drivers/net/ethernet/atheros/alx/main.c
+++ b/drivers/net/ethernet/atheros/alx/main.c
@@ -1618,7 +1618,7 @@ static netdev_tx_t alx_start_xmit(struct sk_buff *skb,
 	return alx_start_xmit_ring(skb, alx_tx_queue_mapping(alx, skb));
 }
 
-static void alx_tx_timeout(struct net_device *dev)
+static void alx_tx_timeout(struct net_device *dev, unsigned int dummy)
 {
 	struct alx_priv *alx = netdev_priv(dev);
 
@@ -2061,7 +2061,7 @@ static pci_ers_result_t alx_pci_error_slot_reset(struct pci_dev *pdev)
 	if (!alx_reset_mac(hw))
 		rc = PCI_ERS_RESULT_RECOVERED;
 out:
-	pci_cleanup_aer_uncorrect_error_status(pdev);
+	pci_aer_clear_nonfatal_status(pdev);
 
 	rtnl_unlock();
 
-- 
2.27.0

